app.rb

```diff
@@ -11,143 +11,66 @@ def initialize()
     @rentals = []
   end
 
-  def app_options
-    puts '1 - List all books'
-    puts '2 - List all people'
-    puts '3 - Create a person'
-    puts '4 - Create a book'
-    puts '5 - Create a rental'
-    puts '6 - List all rentals for a given person id'
-    puts '7 - Exit'.colorize(color: :red).underline
-    puts "\n\n"
-  end
+  @app_options = [
+    '[0]  =>  List all books',
+    '[1]  =>  List all people',
+    '[2]  =>  List all students',
+    '[3]  =>  List all teachers',
+    '[4]  =>  Create a person',
+    '[5]  =>  Create a book',
+    '[6]  =>  Create a rental',
+    '[7]  =>  List all rentals for a given person id',
+    '[8]  =>  List all rentals for a given book',
+    '[9]  =>  Exit'
+  ]
 
   def run
     user_response = 0
     puts "\n\nWelcome to School Library App!\n\n".colorize(color: :green).bold
-    while user_response != '7'
-      puts "Please choose an option by entering a number:\n\n".colorize(color: :magenta)
-      app_options
-      print 'Enter Option [number]: '.colorize(color: :white).bold
+
+    while user_response != '9'
+      puts "Please choose an option by entering a number:\n\n".colorize(color: :magenta).italic
+      @@app_options.each do |choice|
+        if choice.include?('Exit')
+          puts choice.colorize(color: :red)
+        else
+          puts choice
+        end
+      end
+      print "\n\nEnter Option [number]: ".colorize(color: :white).bold
       user_response = gets.chomp
       puts "\n\n"
       check_selection(user_response)
-
     end
-    puts "Thank you for using this app!\n\n".colorize(color: :cyan).bold if user_response == '7'
+    puts "Thank you for using this app!\n\n".colorize(color: :cyan).bold if user_response == '9'
   end
 
   def check_selection(response)
     case response
+    when '0'
+      Book.list_all_books(@books)
     when '1'
-      list_all_books
+      Person.list_all_people(@people)
     when '2'
-      list_all_people
+      Student.lits_all_students(@people)
     when '3'
-      create_person
+      Teacher.list_all_teachers(@people)
     when '4'
-      create_book
+      person = Person.create_person
+      @people << person
+      puts "\n\nPerson created successfully".colorize(color: :green).italic if @people.include?(person)
     when '5'
-      create_rental
+      book = Book.create_book
+      @books << book
+      puts 'Book created successfully' if @books.include?(book)
     when '6'
```

Comment:
-  [ ] I think this it would be better if you placed it in your main app to follow the Single-responsiblity Principle so kindly according to the requirements update it since this options inside the app.rb file

![image](https://user-images.githubusercontent.com/58553711/194976291-168e1928-7d47-4ef3-9f43-87116b1e7fce.png)

Reply:
Hi @Reem-lab, thank you for pointing this out, however, I think you could have added as a suggestion **[OPTIONAL]** :bulb: since my project does meet this requirement. Actually, I had the same comment from a previous reviewer and I asked for a second opinion where the Senior reviewer approved this approach. You can verify this information [in this PR](https://github.com/karlavdelgadof/OOP-school-library/pull/4#:~:text=Project%20is%20working%20well%20when%20running%20the%20main.rb.%20No%20need%20to%20move%20code%20in%20the%20main.rb%20file.)